docker-bake.hcl

```diff
@@ -162,7 +162,8 @@ group "cloudsmith-images" {
   # NOTE: Please keep this list sorted in alphabetical order
   targets = [
     "e2e-tests",
-    "grapl-services"
+    "grapl-services",
+    "rust-integration-tests-new"
```

Comment:
It would probably be better to put the group `integration-tests-new` in here instead of the target `rust-integration-tests-new`.

(Or just remove the `integration-tests-new` group, if it's not serving a clear purpose right now. `integration-tests` is a group because we have Rust and Python, and `e2e-tests` is a bare target because there's only one.)

Reply:
fixed